tests/tests/test_cli.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python
+# Copyright 2017 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        https://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+from common import *
+import pytest
+
+
+class TestMigration:
+    @staticmethod
+    def verify_db_and_collections(client, dbname):
+        dbs = client.database_names()
+        assert dbname in dbs
+
+        colls = client[dbname].collection_names()
+        assert DB_MIGRATION_COLLECTION in colls
+
+    @staticmethod
+    def verify_migration(db, expected_version):
+        major, minor, patch = [int(x) for x in expected_version.split('.')]
+        version = {
+            "version.major": major,
+            "version.minor": minor,
+            "version.patch": patch,
+        }
+
+        mi = db[DB_MIGRATION_COLLECTION].find_one(version)
+        print('found migration:', mi)
+        assert mi
+
+    @staticmethod
+    def verify(cli, mongo, dbname, version):
+        TestMigration.verify_db_and_collections(mongo, dbname)
+        TestMigration.verify_migration(mongo[dbname], version)
+
+
+@pytest.mark.last
```

Comment:
Can you leave a note as to why this must run last?

Reply:
done